arquillian-environment-reporter/arquillian-environment-reporter-impl/src/test/java/org/arquillian/environment/reporter/impl/TestRunnerDetectorTest.java

```diff
@@ -0,0 +1,17 @@
+package org.arquillian.environment.reporter.impl;
+
+import org.junit.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class TestRunnerDetectorTest {
+
+    @Test
+    public void should_return_maven_as_runner() {
+
+        final TestRunnerDetector.TestRunner testRunner = TestRunnerDetector.detect();
+
+        assertThat(testRunner).isEqualTo(TestRunnerDetector.TestRunner.MAVEN);
```

Comment:
This will fail from IDE - shouldn't we have some assumptions about the env and more tests? Or maybe have a `detect()` method getting a stack as an argument instead?